### ISSUE TITLE
Weird flashing when defining "theme-color" meta

### ISSUE THREAD

**Issue by gustavopch** [lordgiotto/react-metatags-hook#3]
Controlling the `<meta name="theme-color" content="...">` with this lib causes a strange flash during the transition from one color to another when you have components like these.
```
const PageWithRedBar = () => {
  useMetaTags({
    metas: [{ name: 'theme-color', content: 'red' }],
  }, [])

  ...
}

const PageWithGreenBar = () => {
  useMetaTags({
    metas: [{ name: 'theme-color', content: 'green' }],
  }, [])

  ...
}
```
When going from PageWithRedBar to PageWithGreenBar, it doesn't simply transition the address bar from red to green. Instead it seems to transition from red to nothing (or the default defined in the actual index.html), then from nothing to green.

**Comment by lordgiotto** [comment]
Hi gustavopch, sorry for the late response.
I'll have a look into it this weekend :)

**Comment by lordgiotto** [comment]
Hey @gustavopch,
today I did some experiments to better understand the issue, and I'm not sure it's related to this library. It looks like that flash happens every time I programmatically change the theme-color.
 
I can be wrong though, so let me explain what I did.
I created a simple react app with:
- a component, mounted as a route on `/red` path, that set `theme-color` meta to **red** using the hook
- a component, mounted as a route on `/green` path, that set `theme-color` meta to **green** using the hook
- an element with a simple onClick function that gets the theme-color meta tag and **manually changes its attribute** to **blue** (without using the hook)
```jsx
<span onClick={() => {
    const themeColor = document.querySelector('meta[name="theme-color"]')
    themeColor.setAttribute('content', 'blue')
}}>MANUALLY CHANGE TO BLUE</span>
```

In the [video attached here](https://github.com/lordgiotto/react-metatags-hook/files/3856129/react-metatags-hook-issue-3.mp4.zip), you can see the result: it looks like that the strange flash during the transition happens every time I change the theme-color, regardless of what I use to change it (the hook or direct manipulation of the dom).

Let me know if I'm missing something and if I can help somehow.

**Comment by gustavopch** [comment]
@lordgiotto Thanks for taking a look at it. I'm watching your video. I don't think the strange flash happens every time you change the theme-color. It only happens in the first time you click "go to red" in your video. Can you confirm it?

**Comment by lordgiotto** [comment]
I can still notice a strange flash every time I change theme-color (it's not a smooth transition between the colors), but for sure it's more noticeable when I there's no theme-color and one is set.
Maybe your issue is related to the absence of a fallback theme-color: consider that the one set in the HTML is not restored by the hook.
Try to add the hook at the higher level possible (like src/App.js) and set a theme-color there: it should provide a fallback so the theme-color meta tag is never removed.
Let me know if it work :)

**Comment by gustavopch** [comment]
@lordgiotto I tried your suggestion. The issue still happens on the first time the hook runs though.

But I've also tried the following experiment:
1. Inspect my mobile browser using Chrome's remote devices feature in the dev tools.
2. Manually change the theme color with the following command using the dev tools console: `document.head.querySelector('meta[name="theme-color"]').setAttribute('content', '#0f0')`
3. The strange flash happens. :(

So I guess react-metatags-hook has nothing to do with this. It may be a bug in Google Chrome.
